examples/jib-multimodule/pom.xml

```diff
@@ -38,7 +38,7 @@
         <plugin>
           <groupId>com.google.cloud.tools</groupId>
           <artifactId>jib-maven-plugin</artifactId>
-          <version>1.0.2</version>
+          <version>1.3.0-SNAPSHOT</version>
```

Comment:
Mark this unresolved when you remove `SNAPSHOT`.

Reply:
Maybe I should make this set from a property like `${jib.maven-plugin-version}`